doc/rados/configuration/filestore-config-ref.rst

```diff
@@ -1,10 +1,15 @@
 ============================
  Filestore Config Reference
 ============================
-.. warning:: Filestore has been deprecated in the Reef release and is no longer supported.
 
-The Filestore back end is no longer the default when creating new OSDs,
-though Filestore OSDs are still supported.
+.. note:: Since the Luminous release of Ceph, Filestore has not been Ceph's
+   default storage back end. Since the Luminous release of Ceph, BlueStore has
+   been Ceph's default storage back end.  However, Filestore OSDs are still
+   supported. See :ref:`OSD Back Ends
```

Comment:
Didn't we decide that as of Reef they aren't supported, as in code is being removed?  Should we instead condense this entire file to directing the gentle reader to redeploy OSDs before upgrading to Reef?
https://github.com/ceph/ceph/pull/50657
https://github.com/ceph/ceph/pull/51682

Reply:
@anthonyeleven: This PR is one is a series that I have labeled "FileStore deprecation". A future pull request (like, a near-future pull request, probably next week if not tomorrow) will hit every place in the documentation suite and communicate the idea that it's time to redeploy OSDs.

Reply:
Would that be soon enough that Reef won't release claiming that Filestore is still supported? To be clear, I'm saying that the ONLY thing to be left in this file for Reef and main should be an admonition to redeploy OSDs before upgrading.

Reply:
> Would that be soon enough that Reef won't release claiming that Filestore is still supported? To be clear, I'm saying that the ONLY thing to be left in this file for Reef and main should be an admonition to redeploy OSDs before upgrading.

Yes, it will be soon enough that Reef will release with documentation that informs readers that Filestore is not supported at all in Reef.

Here's what's going on here: this is (probably) the final commit affecting this file before I have to start keeping to sets of books for (at least) the RADOS docs. One set of future books will be a Reef-(inclusive)-and-future-releases set that mentions Filestore only to warn people away from it and to let them know that Filestore is defunt, and the other set of future books is a Quincy-and-back-in-time-to-possibly-as-far-back-as-Luminous set that still presents Filestore as a supported storage back end. This commit gets the text into good-enough-by-the-standards-of-English grammar shape that I won't have to edit the extant English in both branches of the bifurcation that is about to occur.

There is a similar do-this-here-and-do-that-there issue vis-à-vis cache tiering. Step one: kill as many doubleton birds with one stone as possible. Step two: contend with the surviving singletons.